.deepsource.toml

```
@@ -0,0 +1,7 @@
+version = 1
+
+[[analyzers]]
+name = "go"
```

Isn't enabled missing?
I am not sure 1. if we want to enable it immediately, 2. if what's the default value of `enabled`
```suggestion
name = "go"
enabled = true
```